cmake/modules/AddSwift.cmake

```diff
@@ -707,6 +715,14 @@ function(_add_swift_library_single target name)
       add_dependencies(${target}_IMPLIB ${${target}_IMPORT_LIBRARY})
     endif()
     set_property(TARGET "${target}" PROPERTY NO_SONAME ON)
+  elseif("${SWIFTLIB_SINGLE_SDK}" STREQUAL "ANDROID")
```

Comment:
Could you explain what you mean by "cheesy"? Is something like https://github.com/apple/swift/pull/5068 what you had in mind?